src/app/backend-api.service.ts

```diff
@@ -83,4 +120,31 @@ export class BackendAPIService {
       ReferralHash,
     });
   }
+
+  GetUserDerivedKeys(
+    ownerPublicKey: string
+  ): Observable< { [key: string]: DerivedKey } > {
+    const derivedKeys: { [key: string]: DerivedKey } = {};
+    const req = this.httpClient.post<any>(
+      `${this.endpoint}/get-user-derived-keys`,
+      {
+        PublicKeyBase58Check: ownerPublicKey,
+      },
+    );
+    return req.pipe(
```

Comment:
How does the backend know what your derived keys are? Is there a backend PR I'm missing here?

Reply:
Resolved, right?